include/librealsense2/hpp/rs_options.hpp

```diff
@@ -155,5 +180,97 @@ namespace rs2
     private:
         rs2_options* _options;
     };
+
+    class options_list
+    {
+    public:
+        explicit options_list( std::shared_ptr< rs2_options_list > list )
+            : _list( std::move( list ) )
+        {
+        }
+
+        options_list()
```

Comment:
Needed?

Reply:
Not needed now . Used device_list as an example and used same functions.